autoload/db/adapter/snowflake.vim

```diff
@@ -0,0 +1,44 @@
+if exists('g:autoloaded_db_snowflake')
+  finish
+endif
+let g:autoloaded_db_snowflake = 1
+
+function! s:command_for_url(url) abort
+  let url = db#url#parse(a:url)
+  "extra options here turn off nonsense before/after results
```

Comment:
I don't think we want this for `#interactive()`. I think what you want is to provide a separate `#filter()` function, and add the output modifying options there.

Reply:
I don't have enough experience with DB to know whether this is true. The friendly/timing options show a hello/goodbye message that would show up in interactive mode before and after the actual returned data. Query timing is already handled by dadbod, so I think it might make sense to suppress these. 

Have made the requested change for now, but let me know your thoughts.

Reply:
`#interactive()` is the command used when you call `:DB snowflake://` with no query to start an interactive session. This doesn't have any sort of timing.